### ISSUE TITLE
1.21 OST issue?

### ISSUE THREAD

**Issue by FEFELAND** [Mystogan98/BeatSaviorData#16]
If I play a level from OST V or the FOB pack, notes become fake but damaging after a little bit. Causing me to fail.

If I fail on an OST level, the game locks and I have to close it. (Saw with Magic, Thnks fr th Mmrs and Escape)

Here's the log if it helps at all [_latest.log](https://github.com/Mystogan98/BeatSaviorData/files/8410726/_latest.log)

Mod version: 2.1.11

https://user-images.githubusercontent.com/57236015/161568017-882c617b-fbb8-4a94-b3da-311ed7f581ab.mp4

**Comment by iiedii** [comment]
Experienced the same problem.

**Comment by ChirpyMisha** [comment]
I experienced the same problem when playing the map "Fvrwvrd - I Return". Removing BeatSaviorData gets rid of the problem. 
It gives me an "InvalidOperationException: Sequence contains no elements" from "BeatSaviorData.SwingTranspilerHandler.AddPreswing".

[2022.04.04.21.50.51.log](https://github.com/Mystogan98/BeatSaviorData/files/8412920/2022.04.04.21.50.51.log)
(I started Beat Saber, started the song at 36 seconds in practice mode, failed the level due to the bug, and quit Beat Saber. So I made sure my logs are fairly clean :) )

Edit:
I just found out that it works fine on SliderHead elements. The SliderTail seems to be the issue
